Busobozihakiim/EPICMAIL_api
# Ft delete email #164691903

### Users should be able to delete a single message.

#### What does this PR do?
- Implements the user feature of deleting one message from the inbox using an id

#### Description of Task to be completed?
- With this function, users should be able to delete one message.
- modify the view one message function to accommodate both viewing and deleting of messages
- Write a class method to delete a single email from a list
- write tests for this feature

#### How should this be manually tested?
- Put the link below in your Postman's  URL with HTTP method DELETE
- https://epicmail007.herokuapp.com/api/v1/messages/id where `id` is an integer
- To get this integer open up the URL https://epicmail007.herokuapp.com/api/v1/messages  in another Postman tab which will return all your messages in the format below,
```
{
    "data": [
        {
            "createdOn": "Sat, 23 Mar 2019 16:34:25 GMT",
            "id": 2,
            "message": "String",
            "receiverId": "qw@epictester.com",
            "senderId": "qw@epctester.com",
            "status": "sent",
            "subject": "d"
        },
    ],
    "status": 200
}
```
- It's from this data that you can select the id and put it in place of the <id> and hot the send button
- You should receive the message "Email has been deleted".
- If you don't have any message then you'll have to create one by specifying http method POST and use this link https://epicmail007.herokuapp.com/api/v1/messages with input in the format below 
```
{
  "subject": "String"
  "to": "ValideEmailAddress",
  "from": "ValideEmailaddress,
  "message": "String" 
 }
```
- After creating a message perform the first steps above
- To run tests read [this](https://github.com/Busobozihakiim/EPICMAIL_api/blob/develop/README.md) document

#### Relevant pivotal tracker stories?
- #164691903
